agents/trainer/task.py

```diff
@@ -243,23 +240,22 @@ def gcs_upload(local_dir, gcs_out_dir):
     blob.upload_from_filename(local_file_path)
 
 
-def main(unused_argv):
```

Comment:
I would have thought main() would still need to be main(_) to work with tf.app.run but if this still works then cool.

Reply:
Replaced this with main(_)